Capstone: Lagos Digital Summit 2022

samlorlah opened the pull request:
### Lagos State Digital Summit

- Added index.html was added for the homepage with 5 Sections.
- Added About.html Page to how to mission of the summit.
- Header section has 2 parts which are the logo and the Menu list with a Hover effect
- Added Transition to Mobile Menu Bar.
- Added Style.css for proper styling of the project.
- Added Script.js to dynamically control the page, especially the MORE button in the Speakers Section of the mobile view
- Added Speakers.js to dynamically render the lists of speakers on the DOM.


### Documentation
**Please [Click here](https://www.loom.com/share/316c97bc2f67498d8cbb49a570f94688) to view the recorded video for this project**

samlorlah committed:
complete index and about html file

compile style.css with scss and modified to clear linters

remove all scss file due to linters conflict

add resource images

samlorlah committed:
add and style mobile menu

samlorlah committed:
add js event to hamburger and mobile menu close button

samlorlah committed:
add speaker.js to handle dynamic rendering of featured speakers

add more styles for mobile menu transicton

samlorlah committed:
change font size of desktop menu list

samlorlah committed:
add more speakers for dynamic rendering

add more partners to list of partners

samlorlah committed:
add ReadMe Live Demo link

samlorlah committed:
Linters Errors Corrected

omarsalem7 commented on index.html:
```diff
@@ -8,11 +8,144 @@
 <body>
+    <header>
+        <div class="top-header">
+            <img src="./images/icons/facebook.png" alt="facebook icon">
+            <img src="./images/icons/twitter.png" alt="">
```
- [ ] Kindly fix the image path because the image directory is `Images` not images or change the name of directory to affect in all website instead change of each image path

omarsalem7 commented on README.md:
```diff
@@ -2,16 +2,25 @@
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://samlorlah.github.io/lagos_digital_summit/)
```
- [ ] Kindly fix your demo link because it is not working. It gives me a blank page.

omarsalem7 commented on about.html:
```diff
@@ -0,0 +1,142 @@
+            <img src="./images/capival.png" alt="daum">
+        </div>
+    </section>
+    <footer>
+        <div>
+            <a href="#" class="logo-container">
+                <img src="./images/lasg-coa.png" class="logo-img" alt="Lagos state coat of arm">
+                LASG <span class="logo-sm-text">Summit</span>
+            </a>
+        </div>
+        <div id="copyright">
+            © copyright 2022. All right reserved
+        </div>
+    </footer>
```
- [ ] It will be better if follow the guideline in [behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) because the footer on the about page has a dark background in the desktop version.

omarsalem7 commented on about.html:
```diff
@@ -0,0 +1,142 @@
+            </div>
+        </div>
+    </section>
+    <section class="container-fluid" id="partners-section">
+        <div class="section-header">
+            <h2 id="partners-header">Partners</h2>
+            <span></span>
+        </div>
+        <div id="partners-logos">
+            <img src="./images/flutterwave.png" alt="daum">
+            <img src="./images/airbnb.png" alt="airbnb">
+            <img src="./images/google.png" alt="google">
+            <img src="./images/daum.png" alt="daum">
+            <img src="./images/capival.png" alt="daum">
+        </div>
+    </section>
```
- [ ] Consider making the partner section not displayed on the about page as in Behance design.

omarsalem7 reviewed (changes requested):
Hi @samlorlah 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

![you-are-near](https://media.giphy.com/media/SSR17K6yZWvGCYrfUh/giphy.gif)

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Good presentation demo ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

samlorlah committed:
Create text.txt

samlorlah committed:
Delete images directory

samlorlah committed:
temporary rename from Images to images

samlorlah committed:
Rename from Images to images

samlorlah committed:
rename Images to images

rectify demo page

change footer on about us page to dark background and light text

remove partner section on about desktop page

uwadonat reviewed (approved):
## Hi @Team!

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
![approval](https://user-images.githubusercontent.com/65860678/181524852-3fbdd5ef-3518-4ceb-9289-ece0578d838d.gif)

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

samlorlah committed:
add lasg-coa image to images folder
